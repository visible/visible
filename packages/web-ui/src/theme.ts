export interface Theme {
  foreground: {
    normal: string;
    wash: string;
  };
  background: {
    normal: string;
    wash: string;
  };
  highlight: {
    normal: string;
  };
  border: {
    normal: string;
  };
}
