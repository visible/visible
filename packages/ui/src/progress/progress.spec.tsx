import { render } from '@testing-library/react';
import React from 'react';
import { ThemeProvider } from 'styled-components';

import { theme } from '../theme';
import { Progress } from '.';

describe('Progress', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <ThemeProvider theme={theme}>
        <Progress progress={50} />
      </ThemeProvider>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
