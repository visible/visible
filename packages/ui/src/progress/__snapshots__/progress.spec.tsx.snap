// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress matches snapshot 1`] = `
.c0 {
  position: relative;
}

.c2 {
  position: absolute;
  top: 0;
  left: 0;
  height: 8px;
  -webkit-transition: 0.15s ease-in;
  transition: 0.15s ease-in;
  -webkit-animation: jWuBCz 1.5s ease-in infinite;
  animation: jWuBCz 1.5s ease-in infinite;
  border-radius: 99px;
  background-color: #ffaa01;
}

.c1 {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 8px;
  border-radius: 99px;
  background-color: #e6e6e6;
}

<div
  class="c0"
>
  <div
    class="c1"
  />
  <div
    class="c2"
    style="width: 50%;"
  />
</div>
`;
